motr/ut/idx_dix.c

```diff
@@ -399,12 +416,18 @@ static void ut_dix_namei_ops(bool dist)
 
 static void ut_dix_namei_ops_dist(void)
 {
-	ut_dix_namei_ops(true);
+	ut_dix_namei_ops(true, 0);
+}
+
+static void ut_dix_namei_ops_dist_oi_flags(void)
```

Comment:
may be we can add one more ut, where both oif_skip_layout and oif_crow flag  are set.

Reply:
Added tow more UT's with skip_layout and crow flag.